lib/mof/scanner.rb

```diff
@@ -144,10 +144,10 @@ def fill_queue
 	when "sint64" then @q.push [:DT_SINT64, CIM::Type.new(m)]
 	when "sint8" then @q.push [:DT_SINT8, CIM::Type.new(m)]
```

Comment:
I'm not sure that this issue requires this much complexity. Regular expressions are notoriously hard to manage as they become more complex.  Also I don't see us needing the namespace name and we'd need to modify all of the existing code from `m` to `m.foo`.  This seems like a lot of overhead where a simpler regex and simple string matching would suffice.